### ISSUE TITLE
Refactor  selected sender account/companion account and their balance process.

### ISSUE THREAD

**Issue by hbulgarini** [paritytech/parity-bridges-ui#224]
At the moment , the logic for selected account/companion resides on a react hook at the component level. 

For sake of consistency based on the architecture agreed, all subscriptions should be handled in the data model for the accounts and update states changes through the account context.

In order to keep the changes progressively, this change will try to be split :

- [x] 1) Set companion account and subscribe to accounts balances in the data model.
- [x] 2) Replace `Account` component  in the Sender component for selected account/companion to use presentational component `AccountDisplay` ( currently Account is a wrapper that uses the useBalance hook which handles the subscription there ).
- [x]  3) Create a refresh process that every minute or 30 seconds reads the balances for all the accounts so the sender dropdown does not have to handle subscriptions. Also balances might be updated when the dropdown is opened.

**Comment by hbulgarini** [comment]
Something that i don't have clear yet is if we should replace the balance process for the accounts listed in the dropdown. 
I think that subscribing all the existing accounts to their respective balance might be a bit overwhelming for the node.

We need to think in a way to move this logic to the data model as well but without affecting performance.

**Comment by tomusdrw** [comment]
Balances of accounts in dropdown list are most likely good enough if they are just fetched periodically (say every minute) instead of being subscribed for. We can also do that only if the dropdown is actually open (i.e. after opening the dropdown there might be a small delay until you see balances). Only after the account is selected we should start a subscription to get balance updates as soon as possible.

Ideally (but that's really a stretch goal), we would send batch requests for accounts in groups of 10. (i.e. with 20 accounts we would just send two batch-RPC requests, each asking for 10 accounts - we don't even do that in parallel but rather in the order they appear on the dropdown).

**Comment by hbulgarini** [comment]
> Balances of accounts in dropdown list are most likely good enough if they are just fetched periodically (say every minute) instead of being subscribed for. We can also do that only if the dropdown is actually open (i.e. after opening the dropdown there might be a small delay until you see balances). Only after the account is selected we should start a subscription to get balance updates as soon as possible.
> 
> Ideally (but that's really a stretch goal), we would send batch requests for accounts in groups of 10. (i.e. with 20 accounts we would just send two batch-RPC requests, each asking for 10 accounts - we don't even do that in parallel but rather in the order they appear on the dropdown).

yes, we can create some kind of process that schedules every minute/ 30 seconds the update of all the balances and pre load balances so when the dropdown is opened the user does not see the loading which is something that it is already happening and it looks a little bit weird i think.
I will add a third point to the list in the description :)
